### PR TITLE
Add ACCU

### PR DESCRIPTION
**Pull request by vgarmash**
Add America’s Christian Credit Union which supports MFA

### CODE REVIEW

**Review by stephengroat — CHANGES_REQUESTED** [review]
@vgarmash thanks for the PR. could you add an image?

**Comment by vgarmash** [comment]
I found a logo on Google Play. Converting it to 32x32

**Comment by vgarmash** [comment]
Added logo. Check it out.

**Comment by stephengroat** [comment]
You'll need to properly alphabetize `_data/banking.yml` accord to [travis](https://travis-ci.org/2factorauth/twofactorauth/builds/179552210)

**Comment by vgarmash** [comment]
@stephengroat Can you please help me? After what record should I put ACCU?

**Comment by stephengroat** [comment]
try it before the `America First Credit Union` entry?

**Comment by vgarmash** [comment]
@stephengroat Please check now.

**Comment by vgarmash** [comment]
@stephengroat Actually it worked once I put it after American Express

**Comment by vgarmash** [comment]
I squashed all commits into 1

**Review by stephengroat — APPROVED** [review]

**Comment by stephengroat** [comment]
looks good. Thanks for all the work @vgarmash!

File: _data/banking.yml [hunk 1]
```diff
@@ -42,6 +42,15 @@ websites:
...
       tfa: No
 
+    - name: America’s Christian Credit Union
+      url: https://www.americaschristiancu.com
+      img: accu.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      email: Yes
+      doc: https://www.americaschristiancu.com/business/services/online-banking-bill-pay.html
```
Comment (Jawshy) [inline _data/banking.yml:52]: For me this link is HTTP, not HTTPS.

**Review by Jawshy — CHANGES_REQUESTED** [review]
I followed the provided documentation, but I can't see where they specify the categories of two/multi-factor authentication (I.e. SMS, phone, and email). I can only see that there is 2FA/MFA.

**Comment by vgarmash** [comment]
Yes, in the documentation they do not explain it. Actually MFA is enabled for everyone by default. I registered there today and I got a MFA screen with 3 options and I chosen get code by email and it worked fine. I can give you a screen-shot of it.

File: _data/banking.yml (hunk 1 above)
Comment (vgarmash) [reply _data/banking.yml:52]: Link work any way - it just redirects from https to http...

**Comment by Jawshy** [comment]
> I can give you a screen-shot of it. 

Please do.

**Comment by vgarmash** [comment]
@Jawshy Here it is:
2016-11-28 23_33_56-america s christian cu - opera

**Comment by vgarmash** [comment]
@stephengroat @Jawshy Guys, do you have any other questions?

**Comment by Jawshy** [comment]
That's all from me, @vgarmash. Thanks again.

**Review by Jawshy — APPROVED** [review]

**Pull request merged**